### DISCUSSION TITLE
Non Used Routes

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#3964] [Development Q&A] [answered]
@opencollective/core : Working on https://github.com/opencollective/opencollective-frontend/pull/5855 I realized that the following three routes aren't probably used? 

- confirmEmail.js

- confirm-guest.js

- unsubscribeEmail.js

If so, I was wondering can we remove them along with the route definitions? 🤔

**Comment by Betree** [comment, marked as answer]
All these routes are used, they're just not linked in the frontend. Most are linked from the API's emails. For example, the unsubscribe one: https://github.com/opencollective/opencollective-api/blob/debt/2021-01-29-deprecated-fieds/server/lib/email.js#L263

**Reply by SudharakaP** [reply]
Ah, it didn't missed me that one can use the routes for external API requests etc. Thanks for the clarification. 😄
